README.md

```diff
@@ -16,72 +20,200 @@ make
+```
+
+activate Cekit and run the following command
+
+
+```console
 $ make bundle-dev
+```
+
+the last log line is something like this:
+```console
+INFO  Image built and available under following tags: quay.io/<your_quay_username>/rhpam-operator-bundle:7.12.1, quay.io/${USERNAME}/rhpam-operator-bundle:latest
+```
+###  Push the bundle on the container registry
+
+VERSION=$(go run getversion.go)
+
+USERNAME=<your_quay_username>
+```console
+$ docker push quay.io/${USERNAME}/rhpam-operator-bundle:${VERSION}
+```
+
+### Build the index image
+
+```console
+opm index add --bundles quay.io/${USERNAME}/rhpam-operator-bundle:${VERSION} --tag quay.io/${USERNAME}/rhpam-operator-index:${VERSION}
+```
+### Push the index on the container registry
+
+First time login into your quay account
+```console
+podman login quay.io
+```
+Push the index on your quay repository
+```console
+podman push quay.io/${USERNAME}/rhpam-operator-index:${VERSION}
+```
+
+#### Disable default catalog sources on Openshift
+
+To test your Operator, with bundle and index you need to disable the default source like the operator hub
+```console
+oc patch OperatorHub cluster --type json -p '[{"op": "add", "path": "/spec/disableAllDefaultSources", "value": true}]'
+```
 
+#### Write your Catalog-source
 
-$ docker push ${BUNDLE}
-BUNDLE_DIGEST=$(docker inspect --format='{{index .RepoDigests 0}}' ${BUNDLE})
-INDEX_VERSION=v4.7
-INDEX_IMAGE=quay.io/${USERNAME}/ba-operator-index:${INDEX_VERSION}
-INDEX_FROM=${INDEX_IMAGE}_$(go run getversion.go -csvPrior)
-INDEX_TO=${INDEX_IMAGE}_${VERSION}
+A catalog source is repository of CSVs, CRDs, and packages that define an application.
 
-$ opm index add -c docker --bundles ${BUNDLE_DIGEST} --from-index ${INDEX_FROM} --tag ${INDEX_TO}
+```console
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: xxxxxname
+  namespace: openshift-marketplace
+spec:
+  sourceType: grpc
+  image: xxxxximage
+  displayName: My Operator Catalog
+  publisher: grpc
+```
 
-$ docker push ${INDEX_TO}
+Choose a CATALOG_SOURCE_NAME something like "my-operator-manifests"
 
-# only run in dev env
-$ oc patch operatorhub.config.openshift.io/cluster -p='{"spec":{"disableAllDefaultSources":true}}' --type=merge
-$ oc apply -f - <<EOF
+Example of catalog-source.yaml
+```console
 apiVersion: operators.coreos.com/v1alpha1
 kind: CatalogSource
 metadata:
-  name: my-catalog
+  name: my-operator-manifests
   namespace: openshift-marketplace
 spec:
-  displayName: "Dev Bundles"
-  publisher: "Red Hat"
   sourceType: grpc
-  image: ${INDEX_TO}
-  updateStrategy:
-    registryPoll:
-      interval: 45m
-EOF
+  image: quay.io/<your_quay_id>/rhpam-operator-index:7.12.1
+  displayName: My Operator Catalog
+  publisher: grpc
+```
+
+#### Create catalog source on Openshift
+
+```console
+oc create -f catalog-source.yaml
+```
+
+#### Write your Subscription
+
+A subscription keeps CSVs up to date by tracking a channel in a package.
```

Comment:
instead creating the subscription, wouldn't be better install it using the ocp operator ui?

Reply:
or at least add both options here :)

Reply:
Ok, I've used only bash command because in the readme are used only bash commands